frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -739,6 +739,20 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
     return containers;
   }
 
+  function getInitContainers() {
+    if (!resource) {
+      return [];
+    }
+
+    let initContainers: KubeContainer[] = [];
+
+    if (resource.spec.initContainers) {
+      initContainers = resource.spec.initContainers;
+    }
+
+    return initContainers;
```

Comment:
I think this can be simplified with `return resource?.spec?.initContainers || [];`